databricks/koalas/tests/test_series.py

```diff
@@ -1372,3 +1372,21 @@ def test_take(self):
         self.assertRaises(ValueError, lambda: kser.take("1"))
         self.assertRaises(ValueError, lambda: kser.take({1, 2}))
         self.assertRaises(ValueError, lambda: kser.take({1: None, 2: None}))
+
+    def test_divmod(self):
+        kser = ks.Series([100, None, 300, None, 500], name="Koalas")
+
+        others = [-100, 0, 100, None, np.nan]
+        for other in others:
+            result_div, result_mod = kser.divmod(other)
+            self.assert_eq(repr(result_div), repr(kser.div(other)))
+            self.assert_eq(repr(result_mod), repr(kser.mod(other)))
+
+    def test_rdivmod(self):
+        kser = ks.Series([100, None, 300, None, 500], name="Koalas")
+
+        others = [-100, 0, 100, None, np.nan]
+        for other in others:
+            result_rdiv, result_rmod = kser.rdivmod(other)
+            self.assert_eq(repr(result_rdiv), repr(kser.rdiv(other)))
+            self.assert_eq(repr(result_rmod), repr(kser.rmod(other)))
```

Comment:
Why do we compare against Koalas instead of pandas operations?

Reply:
I think pandas has some bug on those operations.
I'll show you an example a few later after arriving at home.

Reply:
They have different behavior between pandas and Pyspark (which we're using for calculating of `mod`)

```
>>> pdf
   Koalas
0      10

>>> sdf.show()
+------+
|Koalas|
+------+
|    10|
+------+


>>> pdf.mod(-3)
   Koalas
0      -2

>>> sdf.select(sdf.Koalas.__mod__(-3)).show()  # For 10% (-3), pandas returns -2 when PySpark returns 1.
+-------------+
|(Koalas % -3)|
+-------------+
|            1|
+-------------+
```

Reply:
Yup, we should match.

Reply:
@HyukjinKwon 

I made a PR at #1399  with related issue (#1398 )